Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -111,7 +126,7 @@ internal final class CommentsViewController: UITableViewController {
         self?.tableView.reloadData()
       }
 
-    self.viewModel.outputs.configureCommentComposerViewWithData
+    self.viewModel.outputs.commentComposerData
```

Comment:
I don't think we should be observing the same output in two separate places

Reply:
Oh yeah, I must have missed that from merging from master. Thanks for spotting that.